packages/connect-wallet-modal/src/connectButtons/ConnectWC.tsx

```diff
@@ -79,7 +79,7 @@ export const ConnectWC: FC<ConnectWCProps> = (props: ConnectWCProps) => {
         redirectionWindow?.document.write(getRedirectionLoadingHTML());
 
         // Initiate a connection, but do not block the further execution
```

Comment:
Not a fan of this approach. In 1 if path you await connect and in other you skip promise. If you want to continue execution you can:
- save promise to const 
- await in the end of in Promise.all with last promise